src/testing/test-transpile.ts

```diff
@@ -15,7 +15,9 @@ export function transpile(input: string, opts: TranspileOptions = {}): Transpile
     style: null,
     styleImportData: 'queryparams',
     target: 'es2015', // default to es2015
-    transformAliasedImportPaths: parseStencilTranspilePaths(process.env.__STENCIL_TRANSPILE_PATHS__),
+    transformAliasedImportPaths: parseStencilTranspilePaths(
+      process.env.__STENCIL_TRANSPILE_PATHS__ as unknown as string,
```

Comment:
Does this need to be cast as `unknown` first? I think you can get away with just `as string` since the type is already `string | undefined`. Alternatively could just be:

```suggestion
      process.env.__STENCIL_TRANSPILE_PATHS__!,
```

Think they all work fine tho so nbd.